pyanaconda/payload/install_tree_metadata.py

```diff
@@ -236,7 +238,10 @@ def path(self):
         if self.relative_path == ".":
             return self._root_path
         else:
-            return os.path.normpath(os.path.join(self._root_path, self.relative_path))
+            protocol, url = split_protocol(os.path.join(self._root_path, self.relative_path))
+            # resolve problems with relative path especially useful for NFS (root/path/../new_path)
+            url = os.path.normpath(url)
```

Comment:
Actually, do we want to do this if there is something in the protocol?

Reply:
We don't but there is no need to avoid that. This is just simplification instead of adding unnecessary if.

Reply:
OK, I think "URLs won't contain double slashes" is a reasonable assumption, even if not bulletproof.

Reply:
I'm merging this now. If we will find better solution in the future we can update this.